### DISCUSSION TITLE
Crafting Time: Goblin Sapper Charge

### DISCUSSION THREAD

**Discussion by hn1206** [chromiecraft/chromiecraft#4256] [Suggestions]
Hi! 

My suggestion is simple. Reduce the crafting time of Goblin Sapper Charges. 

Right now a Sapper Charge takes 25 seconds to create.

I would suggest a crafting time of maximum 5 seconds.

**Comment by BeefSteaker** [comment]
If crafting time is blizzlike (if not then issues is a better place) , I don't see a reason why one item in particular should be adjusted imo

**Reply by hn1206** [reply]
Fair point. 

The only reason why I wanted to single out this item is due to its 25-second crafting time. I know it is niche, but still a QOL I would like to see :)

**Comment by Amrx2** [comment]
Beef covered this one already, the crafting time is as it should be. If we adjust one then we will end up adjusting them all which is not what we are planning to do.

Appreciate the suggestion either way, have a good day!
Construct
